dns/rfc2136/src/www/services_rfc2136_edit.php

```diff
@@ -149,14 +153,13 @@
                    <td><i class="fa fa-info-circle text-muted"></i> <?=gettext("Interface to monitor");?></td>
                    <td>
                      <select name="interface" class="selectpicker" id="requestif">
- <?php
-                      foreach (get_configured_interface_with_descr() as $if => $ifdesc):?>
+<?php
+                        foreach (get_configured_interface_with_descr() as $if => $ifdesc):?>
```

Comment:
```suggestion
<?php foreach (get_configured_interface_with_descr() as $if => $ifdesc): ?>
```

Reply:
similar for endforeach...

Reply:
I'm sorry I'm really confused about the coding style you are asking for, I thought my changes were matching what I found elsewhere in opnsense php code (eg https://github.com/opnsense/core/blob/master/src/www/index.php).